zookeeper-docs/src/main/resources/markdown/zookeeperAdmin.md

```diff
@@ -1021,16 +1021,9 @@ of servers -- that is, when deploying clusters of servers.
 
 * *electionAlg* :
     (No Java system property)
-    Election implementation to use. A value of "1" corresponds to the
-    non-authenticated UDP-based version of fast leader election, "2"
-    corresponds to the authenticated UDP-based version of fast
-    leader election, and "3" corresponds to TCP-based version of
-    fast leader election. Currently, algorithm 3 is the default.
-    ###### Note
-    >The implementations of leader election 1, and 2 are now
-    **deprecated**. We have the intention
-    of removing them in the next release, at which point only the
-    FastLeaderElection will be available.
```

Comment:
Wouldn't it be a good idea to note that these values have been used in prior versions (note which) and are no longer valid in the current version as they were deprecated in 3.4.0 and removed in...

Something along these lines would help people coming in with issues, ie if they were using 1 or 2 and upgraded.

Reply:
@phunt I have added back the removed explanation of the election algorithm back in the documentation along with changes that you mentioned. Default value was made 3 in 3.2 as per [ZOOKEEPER-499](https://issues.apache.org/jira/browse/ZOOKEEPER-499) and was deprecated in 3.4 and will be removed in 3.6.0.